GliaWidgets/Glia.swift

```diff
@@ -1,35 +1,147 @@
 import UIKit
 import SalemoveSDK
 
+/// Kind of an engagement.
```

Comment:
Ok, good to know. I would personally only update the description itself because it is referred to as `kind` almost everywhere, changing the property name would affect oh so many files.